web-registry/src/hooks/bridge/useBridgable.ts

```diff
@@ -4,33 +4,35 @@ import { TablePaginationParams } from 'web-components/lib/components/table/Actio
 
 import { BatchInfoWithBalance } from 'types/ledger/ecocredit';
 
-// TODO - this hook is a placeholder that right now just simulates a request to
-// the network and an empty response. Use later to implement the data request.
-
-async function stall(stallTime = 3000): Promise<void> {
-  await new Promise(resolve => setTimeout(resolve, stallTime));
-}
+import useEcocredits from 'hooks/useEcocredits';
 
 interface Props {
   address?: string;
   paginationParams?: TablePaginationParams;
 }
 
 interface Output {
-  credits: BatchInfoWithBalance[];
+  bridgableCredits: BatchInfoWithBalance[];
   isLoadingCredits: boolean;
 }
 
 export const useBridgable = ({ address, paginationParams }: Props): Output => {
-  const [credits, setCredits] = useState<BatchInfoWithBalance[]>([]);
-  const [isLoadingCredits, setIsLoadingCredits] = useState(true);
+  const [bridgableCredits, setBridgableCredits] = useState<
+    BatchInfoWithBalance[]
+  >([]);
+  const { credits, isLoadingCredits } = useEcocredits({
+    address,
+    paginationParams,
+  });
 
   useEffect(() => {
-    stall().then(() => {
-      setCredits([]);
-      setIsLoadingCredits(false);
-    });
-  }, []);
-
-  return { credits, isLoadingCredits };
+    if (isLoadingCredits) return;
+    const _bridgableCredits = credits.filter(
+      credit =>
+        credit.classId === 'C03' && Number(credit.balance?.tradableAmount) > 0,
```

Comment:
`C03` in mainnet, but the Toucan credit class will probably be different in redwood. To test, we will have to temporarliy alter this for local and deploy-preview testing. 

Maybe we should set an env variable for this, so it works correctly in redwood?

Reply:
I think this will be C46 in Redwood: 
https://redwood--regen-registry.netlify.app/credit-classes/C46

Reply:
Using an env var now, so deploy preview and redwood should work